wgengine/magicsock/magicsock.go

```diff
@@ -2344,73 +2418,36 @@ func (c *Conn) goroutinesRunningLocked() bool {
 
 func maxIdleBeforeSTUNShutdown() time.Duration {
 	if debugReSTUNStopOnIdle {
-		return time.Minute
+		return 45 * time.Second
```

Comment:
Why 45s?

Reply:
I was impatient. :)  This is only when `TS_DEBUG_RESTUN_STOP_ON_IDLE=1` anyway. I just needed something longer than 30 seconds.